### PR TITLE
Prepares files for project review

### PR DESCRIPTION
**Pull request by VicPeralta**
new file:   README.md
new file:   challenges.js
new file:   index.html
new file:   styles.css

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+<body>
+    <header>
+        <h1 id="title">Mexican food survey</h1>
+        <p id="description"> Welcome to Mexican Food Survey </p>
+    </header>
+    <div class="form-area">
+        <form id="survey-form">
+            <label class="label-text" for="name" id="name-label">Name</label>
+            <input class="input-field" type="text" id="name" placeholder="Enter your name" required>
+            <label class="label-text for=" email" id="email-label">email</label>
```
Comment (VicPeralta) [inline index.html:22]: Please fix this typo:
There is a " missing
<label class="label-text for=" email"
<label class="label-text" for="email" *without the initial space

File: index.html [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+                <label for="red">Red</label>
+                <input type="checkbox" id="both" name="salsa" value="both">
+                <label for="both">Both</label>
+            </div>
+            <br>
+            <label class="label-text" for="comments">Additional comments</label>
+            <textarea name="comments" id="comments" cols="30" rows="10"></textarea>
+            <input type="submit" id="submit" value="submit">
+
+
```
Comment (VicPeralta) [inline index.html:57]: Please remove unnecessary white space as it will make the code cleaner

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+#survey-form{
+    display: flex;
+    flex-direction: column;
+    padding: 20px;
+    margin-bottom: 40px;
+}
+.label-text{
+    padding: 10px 5px;
+    /* border: 2px white solid; */
+    /* border-radius: 15px; */
```
Comment (VicPeralta) [inline styles.css:23]: Please remove commented code, it will make the code cleaner

File: README.md [hunk 4]
```diff
@@ -0,0 +1,2 @@
+# Victor-and-Enoje
+Microverse Coding Challenge Trials
```
Comment (VicPeralta) [inline README.md:2]: Please follow the recommended Readme.md template.
Is important to document your repo professionally, in this way other people can understand faster what is the purpose of your code.

**Review by VicPeralta — COMMENTED** [review]
### Great work!!! 🙂
However, there is some work to do.
Please set the recommended linters for this project, they will help you to format the code properly and give a bunch of suggestions that will make your code for other people easier to understand.

**Review by andersonlebon — APPROVED** [review]
Hi @VicPeralta :wave: , 

### To highlight :champagne: 
- Nice Using inline comments :heavy_check_mark: 
- Nice Using github feature to sumarize your review :champagne: 
- You did great including emojis :+1: 
## Status: Approved 🎉 👏

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Highly Recommended:

- [ ] [OPTIONAL] Please remember to include useful links for documentation to help the student

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**